opennms-doc/guide-admin/src/asciidoc/text/events/eif-adapter.adoc

```diff
@@ -0,0 +1,49 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+[[ga-events-eif]]
+=== Receiving IBM Tivoli Event Integration Facility Events
+
+_OpenNMS_ can be configured to receive _Events_ sent using the https://www.ibm.com/support/knowledgecenter/SSSHTQ_7.3.1/com.ibm.netcool_OMNIbus.doc_7.3.1/omnibus/wip/eifsdk/concept/kaa24487.html[Tivoli Event Integration Facility].
+These EIF events are translated into OpenNMS events using preconfigured rules. The resulting UEI are anchored in the `uei.opennms.org/vendor/IBM/EIF/` namespace, with the name of the EIF https://www.ibm.com/support/knowledgecenter/SSSHTQ_7.3.1/com.ibm.netcool_OMNIbus.doc_7.3.1/omnibus/wip/eifsdk/concept/ecoemst16.html[event class] appended.
+
+A sample event configuration for the OMEGAMON_BASE class is included with OpenNMS.
+
+[[ga-events-eif-configuring]]
+==== Configuring the EIF Adapter
+
+Once _OpenNMS_ is started and the _Karaf_ shell is accessible, you can install the _EIF Adapter_ feature and configure it to listen on a specific interface and port.
+
+NOTE: By default the _EIF Adapter_ is configured to listen on TCP port 1828 on all interfaces.
+
+._OSGi_ login, installation, and configuration of the _EIF Adapter_
+[source]
+----
+[root@localhost /root]# $ ssh -p 8201 admin@localhost
+...
+opennms> features:install eif-adapter
+opennms> config:edit org.opennms.features.eifadapter
```

Comment:
In practice, I had to install `opennms-collection-api` before Karaf would install the `eif-adapter` feature.
I _think_ this is a bug elsewhere, so I did not include it in the documentation here.
`eif-adapter` depends on `opennms-dao-api`, which depends on `opennms-config-api`, which can't resolve a requirement for `org.opennms.netmgt.collection.adapters` until I manually install `opennms-collection-api`, which .. depends on `opennms-config-api` ?